README.md

```diff
@@ -138,6 +138,21 @@ From the root of the project:
 
 Once the sever has started, the webapp is accessible at [https://security-hq.local.dev-gutools.co.uk/](https://security-hq.local.dev-gutools.co.uk/)
 
+### DynamoDB local setup
+
+
```

Comment:
I'd maybe delete the doubled up newlines here and below? No harm though!

Reply:
Ah, I missed that, good catch!